data-service/src/main/java/com/amazonaws/blox/dataservice/handler/EnvironmentHandler.java

```diff
@@ -87,10 +87,10 @@ public EnvironmentVersion createEnvironmentTargetVersion(
     }
   }
 
-  public EnvironmentVersion describeEnvironmentTargetVersion(@NonNull final String environmentId)
+  public EnvironmentRevision describeEnvironmentTargetVersion(@NonNull final String environmentId)
```

Comment:
This signature will become `describeEnvironmentRevision(environmentId, environmentRevisionId)`, then?